tests/labtech/test_lab.py

```diff
@@ -0,0 +1,57 @@
+from pathlib import Path
+from typing import Literal
+
+import labtech
+import pytest
+from labtech.lab import Lab
+
+
+@pytest.fixture()
+def lab(tmp_path: Path) -> Lab:
+    return Lab(storage=tmp_path, max_workers=1)
+
+class TestLab:
+    class TestRunTasks:
+        def test_no_tasks(self, lab: Lab) -> None:
+            lab.run_tasks(tasks=[])
+
+        def test_simple_task(self, lab: Lab) -> None:
+            lab.run_tasks(tasks=[_SimpleTask(a=1)])
+
+        def test_noparam_task(self, lab: Lab) -> None:
+            lab.run_tasks(tasks=[_NoParamTask()])
```

Comment:
Any reason to not check the results while we're running these tests?

```suggestion
        def test_no_tasks(self, lab: Lab) -> None:
            results = lab.run_tasks(tasks=[])
            assert results == {}

        def test_simple_task(self, lab: Lab) -> None:
            results = lab.run_tasks(tasks=[_SimpleTask(a=1)])
            assert results == {_SimpleTask(a=1): 1}

        def test_noparam_task(self, lab: Lab) -> None:
            results = lab.run_tasks(tasks=[_NoParamTask()])
            assert results == {_NoParamTask(): 1}
```

Reply:
Fair suggestion - for me the point of those tests was just checking that the function runs without error - no harm in checking output too. I have committed your suggestion.